### PR TITLE
Recommended channel

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by norkans7** [comment]
@nicpottier please review

File: temba/orgs/models.py [hunk 1]
```diff
@@ -1314,6 +1315,27 @@ def get_export_flows(self, include_archived=False):
...
+            recommended = "nexmo"
+        if countrycode in [country[0] for country in TWILIO_SEARCH_COUNTRIES]:
+            recommended = "twilio"
+        if countrycode == "KE":
+            recommended = "africastalking"
+        if countrycode == "ID":
+            recommended = "hub9"
+        if countrycode == "SO":
+            recommended = "shaqodoon"
+
```
Comment (nicpottier) [inline temba/orgs/models.py]: single quotes here since these are "code" strings not things users will see.

File: temba/utils/__init__.py [hunk 2]
```diff
@@ -359,3 +376,14 @@ def non_atomic_when_eager(view_func):
         return transaction.non_atomic_requests(view_func)
     else:
         return view_func
+
+def timezone_country_code(tz):
```
Comment (nicpottier) [inline temba/utils/__init__.py]: How about ```timezone_to_country_code``` here to clarify what is happening.

**Comment by nicpottier** [comment]
Looks good apert from those two tweaks, nice implementation.

**Pull request merged**